eyal0/valgrind
# Add support for precise computation of SIMD greater-than.

This adds support for 64bit, 16bit, and 8bit to the existing 32bit support.

To test:

```sh
./autogen.sh
./configure
make -j 20
make -j 20 check
perl tests/vg_regtest memcheck/tests/amd64/pcmpgt
perl tests/vg_regtest memcheck/tests/x86/pcmpgt
```

This fixes https://bugs.kde.org/show_bug.cgi?id=432801
